cinder/volume/drivers/vmware/volumeops.py

```diff
@@ -399,6 +402,7 @@ def get_host(self, instance):
                                         self._session.vim, instance,
                                         'runtime.host')
 
+    @cached(cache=TTLCache(maxsize=128, ttl=300))
```

Comment:
Is this safe to cache? Would we try to add a volume to a host if we find it here, even though the host is down already or something?